Documentation/operations/upgrade.rst

```diff
@@ -364,6 +364,11 @@ Agent Options
 * The ``CONNTRACK_LOCAL`` option has been deprecated and will be removed in a
   future release.
 
+Hubble Relay Options
+~~~~~~~~~~~~~~~~~~~~
+
+* The ``hubble.relay.dialTimeout`` helm option has been deprecated and is now a no-op.
```

Comment:
I think this should be under "Helm Options" instead of a separate section, as "Helm Options" already covers Hubble related changes.

Reply:
Yeah agreed, I force-pushed the changes :)